examples/cms/src/App.tsx

```diff
@@ -0,0 +1,64 @@
+import "./App.css"
+
+import { framer, ManagedCollection, ManagedCollectionField } from "framer-plugin"
+import { useEffect, useLayoutEffect, useState } from "react"
+import { DataSource, getDataSource, getDataSourcesIds, syncCollection } from "./data"
+import { FieldMapping } from "./FieldMapping"
+import { SelectDataSource } from "./SelectDataSource"
+import { UI_DEFAULTS } from "./constants"
+
+interface AppProps {
+    collection: ManagedCollection
+    dataSourceId: string | null
+    slugFieldId: string | null
```

Comment:
Something that only becomes clear after reading the code is that `dataSourceId` and `slugFieldId` are the ones previously configured before the plugin was launched. I think we should make that clear in the naming of these props. That will also explain that these will never change at runtime